tools/install_local_wheel.py

```diff
@@ -0,0 +1,88 @@
+
+#! python
+
+import argparse
+import logging
+import os
+import pprint
+from subprocess import call
+import sys
+
+pp = pprint.PrettyPrinter(indent=4, width=100)
+
+def configureLogging(lvl = logging.WARNING, logfile = None):
+    root = logging.getLogger()
+    root.setLevel(lvl)
+
+    formatter = logging.Formatter("[%(asctime)s] [%(levelname)8s] --- %(message)s (%(filename)s:%(funcName)s:%(lineno)s)", "%Y-%m-%d %H:%M:%S")
+    if logfile is None:
+        hndlr = logging.StreamHandler(sys.stdout)
+    else:
+        print("Logging to file %s" % logfile)
+        hndlr = logging.FileHandler(logfile)
+    hndlr.setFormatter(formatter)
+    root.addHandler(hndlr)
+
+def Main():
+    # Setup the required arguments for this script
+    usage = """
+Install the wheel found in bin/<driver>/dist
+"""
+    parser = argparse.ArgumentParser(description=usage)
+    fileGroup = parser.add_argument_group("Input and Output files")
+    fileGroup.add_argument(
+        "--driver",
+        action="store", dest="driver", default=None, required=True,
+        help="Source file")
+
+    verbosityGroup = parser.add_argument_group("Verbosity, Logging & Debugging")
+    verbosityGroup.add_argument(
+        "--exporting",
+        action="store_true", dest="exporting", default=False,
+        help="Are we building an official export? If false, don't actually interact with perforce."
```

Comment:
What is this about Perforce?

Reply:
Removed that option since it doesn't apply here